build.gradle

```diff
@@ -27,7 +27,7 @@ project.ext {
   haleVersion = '4.0.0'
 }
 
-version = '4.0.0'
+version = '4.0.1-SNAPSHOT'
```

Comment:
Rather increase minor than patch version?

Reply:
I mimicked what you did on the `master` branch in earlier releases, e.g. in [this commit](https://github.com/halestudio/hale-cli/commit/75e867aed5f9bccbde4451d7c3826e7f1ff5d49c). You did the minor version increment on the `devel` branch (for which I wanted create another PR after this is merged).

What do you think: should we keep that practice or come up with different approach?

Reply:
Ah, yes. I did not realize this was the `master` branch. Makes sense I think.